### PR TITLE
Article Preview Links

### PR DESCRIPTION
**Pull request by roosnic1**
WPC-61. This PR introduces a new GraphQL mutation that generates a JWT out of the article ID. This token is then combined with a URL that can be shared with externals. The token have a validation period and will not work afterwards.

### CODE REVIEW

File: packages/api/__tests__/utility.ts [hunk 1]
```diff
@@ -33,6 +33,10 @@ class ExampleURLAdapter implements URLAdapter {
   getAuthorURL(author: Author): string {
     return `https://demo.wepublish.ch/author/${author.slug || author.id}`
   }
+
+  getArticlePreviewURL(token: string): string {
+    return `https://demo.wepulish.ch/article/preview/${token}`
```
Comment (penk85) [inline packages/api/__tests__/utility.ts:38]: wepublish -typo

Comment (bishoymelek) [reply packages/api/__tests__/utility.ts:38]: @penk85 sharp eyes :eyes: :smile:

**Review by bishoymelek — APPROVED** [review]
:100: that's great work

**Review by HanaAyman — APPROVED** [review]

**Comment by HanaAyman** [comment]
👍

**Pull request merged**